source/time.tex

```diff
@@ -4771,7 +4771,7 @@
 \begin{itemdescr}
 \pnum
 \returns
-\tcode{year\{int\{x\} + y.count()\}}.
+\tcode{year\{int\{x\} + static_cast<int>(y.count())\}}.
```

Comment:
We need a precondition that `y.count()` is representable by `int` to avoid UB in the conversion. It might be simplest to switch from `\returns: meow` to `\effects: Equivalent to: \tcode{return meow;}`.

Reply:
The conversion is never UB. The addition might, but it could be today as well.

Reply:
Thanks - I forget that everything's twos-complement now. I should probably file an LWG issue for the wider problem of `_Returns_: non-well-defined`; I'm reasonably certain we want implied preconditions, but we don't say so.

Reply:
Just for grins, the recommendation is to stuff this result into a signed 16 bit integral (`short`). :-)  Rationale:  If your civil year number is outside of the range +/-32K, you're hosed anyway.  The civil calendar models the solar system with an accuracy of about 1 day in 4K years.